### ISSUE TITLE
Enhancement: Use project wide preambles, not just current file

### ISSUE THREAD

**Issue by RaphaelArkadyMeyer** [randy3k/LaTeXYZ#9]
In practice, I usually have my preambles stored in their own files like "headers.tex" or "preamble.tex". Out of a bit of frustration with inline math preview, I put together some code to mark a file as preamble via command palette, and automatically include their definition when using inline math in other files. If you are interested, I can make the code cleaner and pull it into the LaTeXYZ repo. Perhaps notably, I achieve this by editing the `preview_math_template_preamble` view setting. I have encountered no consequences for doing this.

**Comment by randy3k** [comment]
I could make a PR. I would consider merging it if it is good enough. Does your code need to iterate all the `include` or `input` statements to search for the commands?

**Comment by randy3k** [comment]
*You could make a PR.

**Comment by RaphaelArkadyMeyer** [comment]
No, instead it requires you to run a function on the command palette for each preamble file you have. In practice, this means once or twice for me, per project.
